group_vars/orcid/main.yml

```diff
@@ -26,9 +26,11 @@ application_host_protocol: 'https'
 application_port: '443'
 project_db_host: '{{postgres_host}}'
 
-# Use Ruby 3.2.0 and install from source
+# Use Ruby 3.2.3 and install from source
 install_ruby_from_source: true
-ruby_version_override: "ruby-3.2.0"
+ruby_version_override: "ruby-3.2.3"
+bundler_version: "2.5.6"
```

Comment:
@bess I do not think we want to be in the business of saying which bundler.  This is what broke the machine. 
```suggestion
```